app/models/concerns/twitter_user/builder.rb

```diff
@@ -0,0 +1,114 @@
+require 'active_support/concern'
+
+module Concerns::TwitterUser::Builder
+  extend ActiveSupport::Concern
+
+  class_methods do
+    def build_by_user(user, attrs = {})
+      build_relation = attrs.has_key?(:build_relation) ? attrs.delete(:build_relation) : false
```

Comment:
__[rubocop]__
* `Hash#has_key?` is deprecated in favor of `Hash#key?`. [:ref](https://github.com/bbatsov/ruby-style-guide#hash-key)
* Line is too long. [94/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)